app/src/ui/changes/changed-file-details.tsx

```diff
@@ -43,18 +42,16 @@ export class ChangedFileDetails extends React.Component<
         <PathLabel path={this.props.path} status={this.props.status} />
         {this.renderDecorator()}
 
-        {enableSideBySideDiffs() && (
```

Comment:
TIL you can do this in react logical render in react.. 😁  It makes total sense; I had just always seen the pattern of call out to a method that would render based on some if.

Reply:
I was more surprised about the `enableSideBySideDiffs() || …` to prevent a component from being rendered 😂